src/sagas/background-refresh.js

```diff
@@ -317,7 +336,6 @@ function* logoutAndCancelScheduler () {
 export default [
   takeEvery(C.START_SCHEDULER_FIXED_DELAY, startSchedulerWithFixedDelay),
   throttle(5000, C.MANUAL_REFRESH, refreshManually),
-  throttle(5000, C.REFRESH_DATA, refreshData),
```

Comment:
This was a (suboptimal) way to merge 3 kinds of refresh sources: manual, time-based and page changes.
Try following scenario:
1. with one page of VMs loaded ( ~ 20 VMS) go to Account Settings
2.  toggle "Slow 3G" network
3. go back and forth quickly using keyboard (few times). Result: each change triggers a fetch. Previously only change that was slow enough triggered the fetch.
4.  stop at one screen. Result: queued fetches are still executed (even for the page that is not currently displayed). Previously only few "late" calls were still executed.